pom.xml

```diff
@@ -65,7 +65,7 @@
   </scm>
 
   <properties>
-    <revision>2.4.2</revision>
+    <revision>2.5</revision>
```

Comment:
Just proposing a version bump. Do we really need to keep the micro version? Can revert if desired.